pages/common/ffe.md

```diff
@@ -0,0 +1,29 @@
+# ffe
+
+> Extract fields from a flat database file and write to another format.
+> `ffe` needs a text config file to interpret the input and format the output.
+> More information: <http://ff-extractor.sourceforge.net/ffe.html>.
+
+- Display all input data using the specified data configuration:
+
+`ffe -c {{path/to/config}} {{path/to/input}}`
+
+- Convert an input file to an output file in a new format:
+
+`ffe -o {{path/to/output}} -c {{path/to/config}} {{path/to/input}}`
+
+- Interpret input with a selected structure and write with a selected print format:
+
+`ffe -s {{structure}} -p {{format}} -c {{path/to/config}} {{path/to/input}}`
+
+- Write only the selected fields:
+
+`ffe -f "{{FirstName,LastNAme,Age}}" -c {{path/to/config}} {{path/to/input}}`
```

Comment:
```suggestion
`ffe -f "{{FirstName,LastName,Age}}" -c {{path/to/config}} {{path/to/input}}`
```

Reply:
I’ll fix the case. The field names are samples matching the data not file names.

Reply:
we use snake case regardless.

Reply:
@CleanMachine1 but we should respect the case used as standard in the target database (e.g. a page for MySQL), I think in this case we don't have one. In the `mongoexport` page we use `field1,field2` so the camelCase used in MongoDB is not affected.

Also, we use snake case but in lower case for placeholders. Wdyt @sbrl ?

Reply:
In this case, we should stick with the existing structure since it looks like it has specific technical meaning. There is a typo here though, which we can fix:

```suggestion
`ffe -f "{{FirstName,LastName,Age}}" -c {{path/to/config}} {{path/to/input}}`
```

If it has already been changed  then there's little point in changing it back, but this is what I would do.

Reply:
@daniel-birket what case do you think is a better fit here?

Reply:
The sample field names in the --file-list sample were from the author's documentation. I purposely departed from the snake_case used required for tokens in your style guide because they were not tokens but a sample of the format required for the parameter. I have since changed it to Cobra_Case as First_Name,Last_Name as suggested, which will serve as well.

Reply:
In this case, let's match the author's documentation convention. This isn't the same as most other token syntax examples, since it's a specific example and not a descriptive stand-in.

Reply:
Ok, I'll change it back to the author's sample format.